docs/user/user/adding-glean-to-your-project/qt.md

```diff
@@ -0,0 +1,107 @@
+# Adding Glean to your Qt/QML project
+
+This page provides a step-by-step guide on how to integrate the [Glean.js](https://github.com/mozilla/glean.js/) library into a Qt/QML project.
+
+Nevertheless this is just one of the required steps for integrating Glean successfully into a project. Check you the full [Glean integration checklist](./index.md) for a comprehensive list of all the steps involved in doing so.
+
+## Requirements
+
+* Python >= 3.6.
+
+## Setting up the dependency
+
+Glean.js' Qt/QML build is distributed as an asset with every Glean.js release. In order to download
+the latest version visit [https://github.com/mozilla/glean.js/releases/latest](https://github.com/mozilla/glean.js/releases/latest).
+
+Glean.js is a [QML module](https://doc.qt.io/qt-5/qtqml-modules-topic.html),
+so extract the contents of the downloaded file wherever you keep your other modules.
+Make sure that whichever directory that module is placed in, is part of the
+[QML Import Path](https://doc.qt.io/qt-5/qtqml-syntax-imports.html#qml-import-path).
+
+After doing that, import Glean like so:
+
+```qml
+import org.mozilla.Glean <version>
+```
+
+The `<version>` number is the version of the release you downloaded minus its patch version.
+For example, if you downloaded Glean.js version `0.15.0` your import statement will be:
+
+```qml
+import org.mozilla.Glean 0.15
+```
+
+## Consuming YAML registry files
+
+Qt/QML projects need to setup metrics and pings code generation manually.
+
+First install the [`glean_parser`](https://mozilla.github.io/glean_parser/) CLI tool.
+
+```bash
+pip install glean_parser
+```
+
+{{#include ../../../shared/blockquote-warning.html}}
+
+### Make sure you have the correct `glean_parser` version!
+
+> Qt/QML support was added to `glean_parser` in version **3.5.0**.
+
+Then call `glean_parser` from the command line:
+
+```bash
+glean_parser translate path/to/metrics.yaml path/to/pings.yaml \
+  -f javascript \
+  -o path/to/generated/files \
+  --option platform=qt \
+  --option version=0.14
```

Comment:
What's this 0.14 about? Is it the Glean version? If so, should it match the version mentioned in the example above?

Reply:
Honestly it can be whatever. It's there just because QML modules require a version number to be imported and the generated folder is a module...

Reply:
https://github.com/mozilla/glean/pull/1667/files#diff-898021a773edca52e54fd0b7ef235d17f5614e0c22186e583899773c342d80e9R72 I do say this below

Reply:
Yeah, this felt a bit confusing. Even if it can be whatever number, maybe let's stick to one version number